astroquery/gaia/core.py

```diff
@@ -579,10 +819,10 @@ def cross_match(self, full_qualified_table_name_a=None,
             raise ValueError("Please, do not specify schema for " +
                              "'results_table_name'")
         query = "SELECT crossmatch_positional(\
-            '"+schemaA+"','"+tableA+"',\
```

Comment:
I find reading strings like this rather difficult, use fstrings, or string formatting instead. Also, long, string lines don't need `\`, but can be wrapped into parens.